iglo/league/utils/paring.py

```diff
@@ -26,18 +26,65 @@ def round_robin(n: int) -> Pairing:
         return list(_round_robin_odd(d, n))
 
 
+def banded_round_robin(player_count: int, band_size: int, add_byes: bool = True) -> Pairing:
+  by_round = []
+  for player_distance in range(band_size, 0, -1):
+    round_2pd_0 = set() # round 2*player_distance
+    round_2pd_1 = set() # round 2*player_distance + 1
+    for player in range(player_count):
+      opponent = player + player_distance
+      if opponent < player_count:
```

Comment:
Why can't we just give a bye in an `else if add_byes` here? we could skip lines from 44 to 58, that additional calculation does not make it clearer what is happening, this if here would.

Reply:
It works for bottom players, but mabye not for the top.
If you have better code I'm happy to take it, I just went for readable add on for byes.